src/utils/api-request.ts

```diff
@@ -166,8 +166,79 @@ export class HttpError extends Error {
   }
 }
 
+/**
+ * Specifies how failing HTTP requests should be retried.
+ */
+export interface RetryConfig {
+  /** Maximum number of times to retry a given request. */
+  maxRetries: number;
+
+  /** HTTP status codes taht should be retried. */
+  statusCodes?: number[];
+
+  /** Low-level I/O error codes that should be retried. */
+  ioErrorCodes?: string[];
+
+  /**
+   * The multiplier for exponential back off. The retry delay is calculated using the formula `(2^n) * backOffFactor`,
```

Comment:
Clarify that it's in seconds
The retry delay is calculated using the formula `(2^n) * backOffFactor` seconds,

Reply:
Done